react/ui-components/src/hoc/app-config-screen-builder/AppFieldComposer.js

```
@@ -0,0 +1,1173 @@
+                  onToggle={onHide}
+                  config={config}
+                />
+              </>
+            ) : (
+              <>
+                <div className="appConfigLabelField-label desc">
+                  <span>{`${t(label)}`}</span>
+                  {Mandatory && <span className="mandatory-span">*</span>}
+                  {helpText && (
+                    <span className="icon-wrapper">
+                      <TooltipWrapper content={t(helpText)} children={<InfoOutline fill={"#C84C0E"} width={"20px"} height={"20px"} />} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)